libs/module-to-standalone/user/shell/src/lib/user-shell.component.ts

```diff
@@ -29,5 +32,5 @@ import { Component, Inject } from '@angular/core';
   },
 })
 export class UserShellComponent {
-  constructor(@Inject(TOKEN) public token: string) {}
+  token = inject(TOKEN);
```

Comment:
One thing I like to do when I create an injectionToken, is to create a custom inject function, like injectToken(). I feel it's nicer.